src/main/java/com/KacFlor/ShopSpring/controller/PromotionController.java

```diff
@@ -48,14 +48,14 @@ public ResponseEntity<?> deleteById(@PathVariable("id") Integer id){
 
     @PreAuthorize("hasAuthority('" + Role.Fields.ADMIN + "')")
     @PatchMapping("/{id}")
-    public ResponseEntity<?> updatePromotion(@RequestBody NewPromotion newPromotion, @PathVariable("id") Integer id){
+    public ResponseEntity<?> update(@RequestBody NewPromotion newPromotion, @PathVariable("id") Integer id){
         promotionService.updatePromotion(newPromotion, id);
         return new ResponseEntity<>(HttpStatus.OK);
     }
 
     @PreAuthorize("hasAuthority('" + Role.Fields.ADMIN + "')")
     @PostMapping("/new")
```

Comment:
Why not just post? Why "new"?

Reply:
because post is for new objects i use patch for update date inside existed objects. That because i use body file NewPromotion same as i use in create new promotions for example and i don't want create another files just for that but i will change name from newPromotion to updatedPromotion

Reply:
done in Fixes branch